### PR TITLE
Improve translations

### PR DESCRIPTION
**Pull request by gergo85**

### CODE REVIEW

**Comment by daftspunk** [comment]
Like this please @gergo85 

```
trans('cms::lang.theme.by_author', ['name' => ...])
```

Do not use `str_replace` since it is built in to the `trans` function.

**Comment by gergo85** [comment]
I made it.

File: modules/system/controllers/updates/_install_plugins.htm [hunk 1]
```diff
@@ -46,7 +45,7 @@ <h4 class="section-header">
                                 </div>
                                 <div class="details">
                                     <h4><?= $plugin['name'] ?></h4>
-                                    <p><?= e(trans('cms::lang.theme.by')) ?> <?= $plugin['author'] ?></p>
+                                    <p><?= trans('cms::lang.theme.by_author', ['name' => $plugin['author']]) ?></p>
```
Comment (daftspunk) [inline modules/system/controllers/updates/_install_plugins.htm]: It should still be escaped with `e()`

File: modules/cms/controllers/themes/_theme_list_item.htm [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <div class="layout-cell min-height theme-description">
     <h3><?= e($theme->getConfigValue('name', $theme->getDirName())) ?></h3>
     <?php if (strlen($author)): ?>
-        <p class="author"><?= str_replace(':name', '<a href="'.e($theme->getConfigValue('homepage', '#')).'">'.e($author).'</a>', e(trans('cms::lang.theme.by_author'))) ?></p>
+        <p class="author"><?= trans('cms::lang.theme.by_author', ['name' => '<a href="'.e($theme->getConfigValue('homepage', '#')).'">'.e($author).'</a>']) ?></p>
```
Comment (LukeTowers) [inline modules/cms/controllers/themes/_theme_list_item.htm]: Don't escape everything individually, do it all at once.
Example:
```php
<?= e(trans('cms::lang.theme.by_author', ['name' => "<a href='" . $theme->getConfigValue('homepage', '#')) . "'>$author</a>"]); ?>
```
Make sure you apply that style to everything in this PR. Don't individually escape data, just do it once.

Comment (daftspunk) [inline modules/cms/controllers/themes/_theme_list_item.htm]: Has to be individual because of the HTML markup I think.

Comment (LukeTowers) [inline modules/cms/controllers/themes/_theme_list_item.htm]: Hmm, you know what, you're right. My bad.

**Comment by gergo85** [comment]
Finally what should I change?

**Comment by daftspunk** [comment]
Thanks for that and great work

**Pull request merged**